src/containers/DataGUI.js

```diff
@@ -0,0 +1,92 @@
+import React, { Component, PropTypes } from 'react';
```

Comment:
> Isn't that necessary to export the class

I don't know what you mean by that but `metafields` component is already exported that's how we import it from other views.

> Pages have the same duplication

Code duplication and code reusing are different things. What I'm trying to say is instead of copy-pasting `MetaFields` code and calling it `DataGUI`, you can reuse the code and just import Metafields in `DataFileEdit` view.

Instead of this;
```javascript
import Metadata from '../DataGUI';
```

Do this;
```javascript
import Metadata from '../MetaFields';
```